src/components/contentful/contentfulOpenLetterForm/contentfulOpenLetterForm.js

```diff
@@ -14,16 +13,16 @@ const axiosConfig = {
 
 const OpenLetterForm = ({ content }) => {
   const {
-    writeALetterButtonText,
-    readYourLetterButtonText,
+    /* writeALetterButtonText,
+    readYourLetterButtonText, */
```

Comment:
Nice catch. These can actually be removed since we switched to using hardcoded translation text instead.